src/code/views/dropdown-view.coffee

```diff
@@ -46,7 +54,41 @@ module.exports = React.createClass
       )
       (div {className: menuClass, onMouseLeave: @blur, onMouseEnter: @unblur},
         (ul {},
-          (DropdownItem {key: item.name, item: item, select: @select}) for item in @props.items
+          (DropdownItem {name: item.name or item, item: item, select: @select, isActionMenu: @props.isActionMenu}) for item in @props.items
```

Comment:
It looks like you're right about that. I changed it to `name` because I wanted to *use* that attribute, but when you set `key` on a component it turns out it's not available at `@props.key`. Instead of figuring out why, I chose to use `name`.

So if I want to pass in a name I'll have to pass *both* key and name. Or, I guess, just re-calculate name in the view.

I will update this now.